model/riscv_sys_control.sail

```diff
@@ -74,6 +88,30 @@ function is_CSR_defined (csr : csreg, p : Privilege) -> bool =
     0x143 => haveSupMode() & (p == Machine | p == Supervisor), // stval
     0x144 => haveSupMode() & (p == Machine | p == Supervisor), // sip
 
+    /* supervisor mode: Machine Control Transfer Records Control Register */
+    0x14E => haveSsctr() & (p == Machine | (p == Supervisor & isSsctrEnabled())), // sctrctl
+    /* supervisor mode: Machine Control Transfer Records Status Register */
+    0x14F => haveSsctr() & (p == Machine | (p == Supervisor & isSsctrEnabled())), // sctrstatus
+
+    /* supervisor mode: indirect register select and alias CSRs */
+    0x150 => haveSupMode() & haveSscsrind() &
+             (p == Machine | (p == Supervisor & isSscsrindEnabled())), // siselect
+    0x151 => haveSupMode() & haveSscsrind() & isValidSiselect() &
+             (p == Machine | (p == Supervisor & isSscsrindEnabled())), // sireg
+    0x152 => haveSupMode() & haveSscsrind() & isValidSiselect() &
+             (p == Machine | (p == Supervisor & isSscsrindEnabled())), // sireg2
+    0x153 => haveSupMode() & haveSscsrind() & isValidSiselect() &
+             (p == Machine | (p == Supervisor & isSscsrindEnabled())), // sireg3
+    0x155 => haveSupMode() & haveSscsrind() & isValidSiselect() &
+             (p == Machine | (p == Supervisor & isSscsrindEnabled())), // sireg4
+    0x156 => haveSupMode() & haveSscsrind() & isValidSiselect() &
+             (p == Machine | (p == Supervisor & isSscsrindEnabled())), // sireg5
+    0x157 => haveSupMode() & haveSscsrind() & isValidSiselect() &
+             (p == Machine | (p == Supervisor & isSscsrindEnabled())), // sireg6
+
+    /* supervisor mode: Machine Control Transfer Records Depth Register */
+    0x15F => haveSsctr() & (p == Machine | (p == Supervisor & isSsctrEnabled())), // sctrdepth
```

Comment:
I think the sctr* CSRs above should use haveSmctr() rather than haveSsctr()

Reply:
We can just remove Ssctr everywhere since Smctr cannot be implemented without Ssctr and vice versa.

Reply:
updated

Reply:
still see haveSsctr?